packages/linux/changelog.yml

```
@@ -1,4 +1,12 @@
 # newer versions go on top
+- version: "0.6.2"
```

Btw, is this package used by default on linux agents? If that is the case, should it be GA'ed? (This can be left for a different discussion/PR in any case).
```suggestion
- version: "1.0.0"
```

---

Nope, it's not installed by default.